src/libraries/System.Private.CoreLib/src/System/Number.Formatting.cs

```diff
@@ -1579,7 +1572,7 @@ internal static unsafe void WriteTwoDigits<TChar>(uint value, TChar* ptr) where
 
             Unsafe.CopyBlockUnaligned(
```

Comment:
This is basically just doing span.CopyTo, right?

Is the codegen significantly different if we switch to the safe version? Should we log an issue if there's some places that can't be constant folded down here?

Reply:
@tannergooding yep, that was my initial intent, but looks like for that we need to change too many methods from `TChar*` to `Span`. My initial attempt to do so had a terrible perf impact so I decided to file a general clean up first.

> Should we log an issue 

I guess we can specifically for this, but it all part of the general issue https://github.com/dotnet/runtime/issues/94941

Reply:
The suggestion of logging an issue was mainly for the consideration of if `Unsafe.CopyBlockUnaligned` was slower than `Span.CopyTo`

Is it worth doing `new Span(ptr, length).CopyTo(dest, length)` in the meantime which should get us slightly better and then we can track the switch to use span everywhere separately?